agcaballero/portfolio/src/main/webapp/contact.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <title>Contact Me</title>
+    <script src="https://smtpjs.com/v3/smtp.js"></script>
     <link rel="stylesheet" href="style.css" />
```

Comment:
Is this actually used? It looks like the form's action is currently just a mailto link. If it is used now or in the future we need to be careful about usage restrictions, etc.

Reply:
I was originally considering using it; however, now I don't believe I will. I'll remove the line.

Reply:
Done